jsonnet/telemeter/client/telemeter-client.libsonnet

```diff
@@ -0,0 +1,11 @@
+(import 'kubernetes/kubernetes.libsonnet') + {
+  _config+:: {
+    namespace: 'default',
+
+    telemeterClientSelector: 'job="telemeter-client"',
```

Comment:
as long as we have everything under control for this I don't think we need to make this configurable

Reply:
got it. I was trying to follow the conventions in kube-prometheus but think about this detail